frontend/package.json

```diff
@@ -27,26 +27,27 @@
     "format": "prettier --write \"./**/*.ts\" \"./**/*.tsx\" \"./**/*.scss\"",
     "lint:goal": "eslint --config ./.eslintrc.goal.js --ext .json,.js,.ts,.jsx,.tsx ./src ",
     "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js",
-    "test": "run-s test:lint test:unit test:type-check test:integration-ci",
+    "test": "run-s test:lint test:type-check test:unit test:integration-ci test:cypress-ci",
     "test:dev": "run-s test:lint test:unit test:accessibility test:integration",
     "test:fix": "eslint --ext .js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .js,.ts,.jsx,.tsx ./src",
     "test:type-check": "tsc --noEmit",
     "test:jest": "jest",
     "test:unit": "npm run test:jest",
     "test:accessibility": "test-storybook --verbose",
+    "test:cypress-ci": "npx concurrently -k -s first \"npm run cypress:server:build && npm run cypress:server\" \"npx wait-on tcp:127.0.0.1:9001 && npm run cypress:run:mock\"",
     "test:integration": "playwright test ./src/__tests__/integration",
     "test:integration-ci": "npx concurrently -k -s first -n \"SB,TEST\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:accessibility && npm run test:integration\"",
     "test:e2e": "export E2E=1; playwright test ./src/__tests__/e2e",
     "storybook": "storybook dev -p 6006 -s ../config",
     "cypress:open": "cypress open --project src/__tests__/cypress",
-    "cypress:run": "cypress run -b chrome --project src/__tests__/cypress && npm run cypress:format",
+    "cypress:run": "cypress run -b chrome --project src/__tests__/cypress",
     "cypress:open:mock": "MOCK=1 npm run cypress:open",
     "cypress:run:mock": "MOCK=1 npm run cypress:run",
     "cypress:open:record": "RECORD=1 npm run cypress:open",
-    "cypress:run:record": "RECORD=1 npm run cypress:run",
+    "cypress:run:record": "RECORD=1 npm run cypress:run && npm run cypress:format",
```

Comment:
We only need to format after recording. 
Recording snapshots is an experimental feature that's not being used right now.